README.md

```diff
@@ -2,6 +2,38 @@
 
 A pilot project to extract medical indications (when a compound treats a disease) from the literature using [Snorkel](https://github.com/HazyResearch/snorkel).
 
+##Installing/Setting Up The Conda Environment
```

Comment:
Great work with the README and documentation.

Minor markdown comments (stylistic not functional).

+ space after heading pounds: `## Installing/Setting Up The Conda Environment`
+ blank line after heading